app/services/waste_exemptions_engine/generate_renew_token_service.rb

```diff
@@ -1,40 +1,11 @@
 # frozen_string_literal: true
 
-require_relative "concerns/can_process_jwt_tokens"
-
 module WasteExemptionsEngine
   class GenerateRenewTokenService < BaseService
-    include CanProcessJwtTokens
-
     def run(registration:)
-      payload = {
-        data: {
-          registration_id: registration.id
-        },
-        exp: exp
-      }
-
-      registration.renew_token = generate_token(payload)
+      registration.regenerate_renew_token
```

Comment:
@irisfaraway I will leave with you the decision of keeping this service or get rid of it. It feels quite unnecessary to me now :D

Reply:
I'd vote for deleting it.